src/game.js

```
@@ -16,11 +17,23 @@ class Game {
 		dbGame.push(game);
 		game.session = `http://localhost:3000/game?token=${token}`;
 
-		return Promise.resolve({
-			id : game.id, 
-			session : game.session,
-			playerId : game.playerId
-		})
+		return GameDataController.createGame({			
+				Token : token,
+				PlayerId : game.playerId
+			})
+			.then(async (res) => {
+				return Promise.resolve({
+					id : res, 
+					session : game.session,
+					playerId : game.playerId
+				})				
+			}).catch(err => {				
+				resolve({

```

same here, don't swallow the exception

---

the return form was replaced